src/vehicle.h

```diff
@@ -891,6 +903,18 @@ class vehicle
      */
     float wheel_area( bool boat ) const;
 
+    /**
+     * Calculates the average radius of the wheels of the vehicle.
+     * @param boat If true, calculates the area under "wheels" that allow swimming.
+     */
+    float wheel_radius( bool boat ) const;
+
+    /**
+     * Calculates the mass of the wheels of the vehicle.
+     * @param boat If true, calculates the area under "wheels" that allow swimming.
+     */
+    float wheel_mass( bool boat ) const;
```

Comment:
`wheel_mass( true )` doesn't seem to be utilized. Boolean flags are best avoided wherever possible, especially in case of a single argument. They make reading harder (one needs to see the declaration to learn the meaning) and clearly state that a function does more than one thing.